BP/history/provinces/bp_k_dalmatia.txt

```diff
@@ -1,5 +1,16 @@
-﻿#k_dalmatia
+﻿#Notes:
+# Latest possible attestation of Illyrian is the 6th century, more definitely 4th-5th Century (St. Jerome)
+# https://en.wikipedia.org/wiki/Illyrian_language#Era
+#
+# So parts of Illyria, especially the mountains, were still likely tribal and pagan until the 6th century
+# when the Slavs invaded. Probably played a role in the evolution of the Bosnian Church
```

Comment:
Ok, but Krajina is not mountainous?

Reply:
Right; https://imperium.ahlfeldt.se/ lists very few Roman settlements on the Una/Oeneus river in that region and none of any size (especially in that region), so I presumed it was not colonized much at all. Hence why it has a tribal settlement as well.

Reply:
Ok